src/widgets/checkbox.rs

```diff
@@ -18,17 +18,41 @@ fn checkbox_svg(checked: ReadSignal<bool>) -> impl View {
 
 /// Renders a checkbox the provided checked signal.
 /// Can be combined with a label and a stack with a click event (as in `examples/widget-gallery`).
-pub fn checkbox(checked: ReadSignal<bool>) -> impl View {
-    checkbox_svg(checked).keyboard_navigatable()
+pub fn checkbox(checked: impl Fn() -> bool + 'static) -> ValueContainer<bool> {
+    labeled_checkbox(checked, || "")
```

Comment:
This is not quite the same as before. Here's the new version:

![image](https://github.com/lapce/floem/assets/1407980/56510a55-5b0a-4356-95fe-d6716914735b)

I'd consider keeping the separate implementation.